translations/messages.en.yaml

```diff
@@ -57,8 +57,10 @@ cover: Cover
 related_posts: Related posts
 random_posts: Random posts
 federated_magazine_info: This magazine is from a federated server and may be incomplete.
+disconnected_magazine_info: This magazine is not receiving updates (last activity %days% day(s) ago). Subscribe to start receiving updates.
+always_disconnected_magazine_info: This magazine is not receiving updates. Subscribe to start receiving updates.
 federated_user_info: This profile is from a federated server and may be incomplete.
-go_to_original_instance: Browse more on the original instance.
+go_to_original_instance: For a complete list of posts, visit the original instance.
```

Comment:
it can be posts.. but also threads, right?

Reply:
I think in this case "posts" is alright. In our UI it is `Threads` and `Microblogs` so posts can count as both

Reply:
Sure ok.